idlgen/hbs/composite.hbs

```diff
@@ -1,5 +1,5 @@
-type {{{lookup @root/type_names id}}} = record {
+type {{{lookup @root/type_names id}}} = struct {
```

Comment:
We might as well change `variant` to `enum` to keep it consistent with Rust

Reply:
agreed